pkg/messaging/rabbitmq/setup_test.go

```diff
@@ -11,15 +11,20 @@ import (
 	"syscall"
 	"testing"
 
-	"github.com/mainflux/mainflux/logger"
+	"github.com/gogo/protobuf/proto"
+	logg "github.com/mainflux/mainflux/logger"
```

Comment:
I do not like this `logg`. I would use `mflogger` or something like this.

Reply:
I had updated to mainflux_log, same as mqtt PR.
but Nats testsubscribe is failing, so I haven't pushed yet